src/BlazorUI/Bit.BlazorUI.Extras/Styles/extra-components.scss

```
@@ -4,6 +4,7 @@
 @import "../Components/DataGrid/Pagination/BitDataGridPaginator.scss";
 @import "../Components/ErrorBoundary/BitErrorBoundary.scss";
 @import "../Components/Flag/BitFlag.scss";
+@import "../Components/FullCalendar/BitFullCalendar.scss";
```

Stylelint (17.13.0)

[error] Unexpected extension ".scss" in `@import` (scss/load-partial-extension)

(scss/load-partial-extension)